packages/vue-components/src/panels/MinimalPanel.vue

```diff
@@ -93,6 +103,7 @@ import retriever from '../Retriever.vue';
 import panelBase from './PanelBase';
 
 export default {
+  inheritAttrs: false,
```

Comment:
hmm what's this for?

Reply:
[Vue passes non-prop attributes such as `id` to the root node of a component](https://v3.vuejs.org/guide/component-attrs.html#non-prop-attributes). In this case, we do not want the main panel container, `div` or `span`, to have the `id` as this interferes with anchor navigation.
The `inheriAttrs: false` is needed to prevent the root node of `minimalPanel` and `nestedPanel` from inheriting the `id` attribute. 

https://v3.vuejs.org/guide/component-attrs.html#disabling-attribute-inheritance

Reply:
Thanks for clarifying! Hmm how about changing the property name under `assignPanelId` (then using it as a prop)?

So there's no risk of side effects from disabling attribute inheritance entirely.

Reply:
Yup that would work as well. Will update it.